x/ethbridge/keeper/keeper.go

```diff
@@ -104,30 +104,31 @@ func (k Keeper) ProcessBurn(ctx sdk.Context, cosmosSender sdk.AccAddress, msg *t
 	logger := k.Logger(ctx)
 	var coins sdk.Coins
 
-	if msg.Symbol == types.CethSymbol {
-		coins = sdk.NewCoins(sdk.NewCoin(types.CethSymbol, msg.CethAmount.Add(msg.Amount)))
-	} else {
-		coins = sdk.NewCoins(sdk.NewCoin(msg.Symbol, msg.Amount), sdk.NewCoin(types.CethSymbol, msg.CethAmount))
-	}
-
-	err := k.bankKeeper.SendCoinsFromAccountToModule(
-		ctx, cosmosSender, types.ModuleName, coins,
-	)
-
-	if err != nil {
-		logger.Error("failed to send coin from account to module.",
-			errorMessageKey, err.Error())
-		return err
-	}
-
 	if k.IsCethReceiverAccountSet(ctx) {
 		coins = sdk.NewCoins(sdk.NewCoin(types.CethSymbol, msg.CethAmount))
-		err = k.bankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, k.GetCethReceiverAccount(ctx), coins)
+
+		err := k.bankKeeper.SendCoins(ctx, cosmosSender, k.GetCethReceiverAccount(ctx), coins)
 		if err != nil {
-			logger.Error("failed to send ceth from module to account.",
+			logger.Error("failed to send ceth from account to account.",
```

Comment:
I know there could be security implications, but could this provide more details about each "account"? Otherwise it may not be very helpful as a log msg

Reply:
This is helpful context to log if there is a message. We will be able to find out what tx it came from in the sifnoded logs.